spec/builtin-kill.test.sh

```diff
@@ -0,0 +1,28 @@
+## oils_failures_allowed: 2
+## compare_shells: dash bash-4.4 mksh zsh
```

Comment:
Please change this to 'bash' (to specify bash 5), since bash-4.4 is only for old tests